bug(dashboard): failed to display mv on source in stream graph page

### Describe the bug

When MV on source exists, the stream graph page will crash:
<img width="1244" alt="image" src="https://user-images.githubusercontent.com/14160025/205061086-bb132382-bb7d-487f-be1b-20073f42dba4.png">

error info:
```
main-deecab77a43dfa56.js:1 Error: no such id 1003
```
1003 is the source id.

The dependent relations could be source, recursive construction of the mv graph will fail when it hits the source relation. We should filter non-mv relations when build streaming graph.


### To Reproduce

_No response_

### Expected behavior

_No response_

### Additional context

_No response_

--- a/dashboard/pages/streaming_graph.tsx
+++ b/dashboard/pages/streaming_graph.tsx
@@ -31,12 +31,13 @@ const SIDEBAR_WIDTH = "200px"
 
 function buildMvDependencyAsEdges(mvList: RwTable[]): ActorPoint[] {
   const edges = []
+  const mvSet = new Set(mvList.map((mv) => mv.id))
   for (const mv of reverse(sortBy(mvList, "id"))) {
     if (!mv.name.startsWith("__")) {
       edges.push({
         id: mv.id.toString(),
         name: mv.name,
-        parentIds: mv.dependentRelations.map((r) => r.toString()),
+        parentIds: mv.dependentRelations.filter((r) => mvSet.has(r)).map((r) => r.toString()),
         order: mv.id,
       })
     }

--- a/dashboard/pages/streaming_graph.tsx
+++ b/dashboard/pages/streaming_graph.tsx
@@ -37,7 +37,9 @@ function buildMvDependencyAsEdges(mvList: RwTable[]): ActorPoint[] {
       edges.push({
         id: mv.id.toString(),
         name: mv.name,
-        parentIds: mv.dependentRelations.filter((r) => mvSet.has(r)).map((r) => r.toString()),
+        parentIds: mv.dependentRelations
+          .filter((r) => mvSet.has(r))
+          .map((r) => r.toString()),
         order: mv.id,
       })
     }

